src/Common/CliFolderPathCalculatorCore.cs

```diff
@@ -1,11 +1,9 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-
-//only Microsoft.DotNet.MSBuildSdkResolver (net7.0) has nullables enabled
-#pragma warning disable IDE0240 // Remove redundant nullable directive
+#pragma warning disable IDE0240
 #nullable enable
```

Comment:
shouldn't this just remove the `#nullable enable`

Reply:
You cannot. It is a shared file. You'd have to do the entire Microsoft.NET.Build.Tasks and (likely) the dotnet CLI project, which are thousands of changes.

Reply:
I've readded the original comments so this file no longer has changes in this PR.